webapp/src/marked.tsx

```diff
@@ -287,6 +287,16 @@ export class MarkedContent extends data.Component<MarkedContentProps, MarkedCont
 
 
     private renderVideo(content: HTMLElement){
+
+        pxt.Util.toArray(content.querySelectorAll('iframe.yt-embed'))
+        .forEach((inlineVideo: HTMLElement)=>{
+
+            let lang=pxt.appTarget.appTheme?.defaultLocale ?? "en";
+            let src=content.querySelectorAll('iframe.yt-embed')[0].getAttribute('src');
+            content.querySelectorAll('iframe.yt-embed')[0].setAttribute('src',src+"?hl="+lang);
```

Comment:
I think you were relying on passing the start / end time as query parameters already, which would not work with this -- you'd build a url like

`youtube/embed/blah?startTime=1&endTime=3?hl=zh-CN`

-- maybe https://stackoverflow.com/a/44160941 will be helpful to look at~

Reply:
i think it's fixed now, thanks for helpful reference!